src/openforms/authentication/models.py

```diff
@@ -290,7 +296,7 @@ def to_auth_context_data(
             # EHerkenning without machtigen/mandate
             # Presence of an acting subject confirms eHerkenning (or eidas? TODO),
             # as opposed to a registrator entering the KVK number.
-            case (AuthAttribute.kvk, "") if self.acting_subject_identifier_value:
+            case (AuthAttribute.kvk, "") if not has_registrator:
```

Comment:
Why not:

```suggestion
            case (AuthAttribute.kvk, "") if not submission.has_registrator:
```

Reply:
completely missed that property exists :-) will update!